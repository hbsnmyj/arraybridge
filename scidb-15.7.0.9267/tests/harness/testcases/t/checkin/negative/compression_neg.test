--setup
--test

--error --code=scidb::SCIDB_SE_SYNTAX::SCIDB_LE_COMPRESSOR_DOESNT_EXIST "CREATE ARRAY test < attr_one : float compression 'does_not_exist' > [ I=0:9,10,0 ]"

--cleanup
