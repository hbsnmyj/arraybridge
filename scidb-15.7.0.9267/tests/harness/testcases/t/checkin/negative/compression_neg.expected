[An error expected at this place for the query "CREATE ARRAY test < attr_one : float compression 'does_not_exist' > [ I=0:9,10,0 ]". And it failed with error code = scidb::SCIDB_SE_SYNTAX::SCIDB_LE_COMPRESSOR_DOESNT_EXIST. Expected error code = scidb::SCIDB_SE_SYNTAX::SCIDB_LE_COMPRESSOR_DOESNT_EXIST.]

